### PR TITLE
created network tests for the network manager

### PR DESCRIPTION
**Pull request by k-code00**
created network tests for the network manager

### CODE REVIEW

File: DeBlockWallet.xcodeproj/xcuserdata/consultant.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist
Comment (mattmaloof) [inline DeBlockWallet.xcodeproj/xcuserdata/consultant.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: This file is not needed / local files not needed.

File: DeBlockWallet.xcodeproj/project.xcworkspace/xcuserdata/consultant.xcuserdatad/UserInterfaceState.xcuserstate
Comment (mattmaloof) [inline DeBlockWallet.xcodeproj/project.xcworkspace/xcuserdata/consultant.xcuserdatad/UserInterfaceState.xcuserstate:1]: This file is not needed / local files not needed.

**Review by mattmaloof — APPROVED** [review]
rest looks fine

File: DeBlockWalletTests/NetworkTests.swift
Comment (gisele-prog) [inline DeBlockWalletTests/NetworkTests.swift:1]: tests look fine!

**Review by gisele-prog — APPROVED** [review]

**Pull request merged**